storage/EtcCrypttab.h

```diff
@@ -199,6 +199,10 @@ namespace storage
          * Dump the current contents to the log.
          **/
         void log();
+
+    private:
+
+	static const int DEFAULT_PERMISSIONS = 0640;
```

Comment:
btw can you comment in code why 0640 and not 0600?

Reply:
You are right, it should be 0600.